PC/pyconfig.h

```diff
@@ -307,13 +307,19 @@ Py_NO_ENABLE_SHARED to find out.  Also support MS_NO_COREDLL for b/w compat */
                         /* So MSVC users need not specify the .lib
                         file in their Makefile (other compilers are
                         generally taken care of by distutils.) */
+#                       if defined(Py_LIMITED_API)
+#                               define PINNED_VER "3"
+#                       else
+#                               define PINNED_VER "313"
```

Comment:
I'm not sure exactly how the release manager finds these variables to update them, but we might need to update some of their docs/tools for this change.